src/test/tools/convert_xml_to_yaml/xml2yaml.yaml

```diff
@@ -0,0 +1,40 @@
+host:
+- id: server
+  process:
+  - arguments: tgen.server.graphml.xml
+    plugin: tgen
+    starttime: 1
+- id: client
+  process:
+  - arguments: tgen.client.graphml.xml
+    plugin: tgen
+    starttime: 2
+option:
+  stoptime: 3600
+plugin:
```

Comment:
Would be nice if we could match the typical order we use in xml files. In particular would be good to have the topology and the plugin definitions before the hosts, since those are both referenced from within the host process objects.

I think either hard-coding an order matching our "typical" order in the examples, or preserving the order in whatever source xml file being translated would be fine; whichever is easier.

Reply:
Should be better :)